src/v/cluster/node/local_monitor.cc

```diff
@@ -131,12 +131,14 @@ void local_monitor::update_alert_state() {
         auto [min_by_bytes, min_by_percent] = minimum_free_by_bytes_and_percent(
           d.total);
         auto min_space = std::min(min_by_percent, min_by_bytes);
-        clusterlog.debug(
-          "min by % {}, min bytes {}, disk.free {} -> alert {}",
-          min_by_percent,
-          min_by_bytes,
-          d.free,
-          d.free <= min_space);
+        if (clusterlog.is_enabled(ss::log_level::debug)) {
+            clusterlog.debug(
+              "min by % {}, min bytes {}, disk.free {} -> alert {}",
+              min_by_percent,
+              min_by_bytes,
+              d.free,
+              d.free <= min_space);
+        }
```

Comment:
what computation is being avoided? if you are only thinking about the `d.free <= min_space` expression, did you consider that `is_enabled` is also loading a value from a `std::atomic`?

Reply:
I didn't think this was necessary here.. just addressing your [review comment](https://github.com/redpanda-data/redpanda/pull/3715#discussion_r812385717)--though the code has changed a bit.

Reply:
The basic problem with this adding an `log.is_disabled` check is the values logged are needed for other things.

Reply:
Before it looked like floating point operations were done to create the parameters to log which would then be thrown again if the log level wasn't done. In this formulation the values are already available, so I don't think there is any value in checking if the log level is enabled.